src/test/java/uk/gov/hmcts/reform/sscs/service/AuthorisationServiceTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import feign.FeignException;
+import feign.Request;
+import feign.RequestTemplate;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.sscs.exception.AuthorisationException;
+import uk.gov.hmcts.reform.sscs.exception.InvalidHeaderException;
+
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+@AutoConfigureMockMvc
+@RunWith(MockitoJUnitRunner.class)
+class AuthorisationServiceTest {
+
+    private static final String AUTHORIZATION = "Bearer eyJhbGciOiJIUzUxMiL7.eyJzdWIiOiJzc2NzIiwiZXhwIjoxNjQ2NDA5MjM5fQ"
+            + ".zSEbvMJedOGo16yBOXecLgucWyavnoVu023cterreUF0sxPlmV-Qu8Y7OloJUKrLGlNweUr8mVpYWPzE0iNyYw";
+    private static final String BAD_AUTHORIZATION = "eyJhbGciOiJIUzUxMiL7.eyJzdWIiOiJzc2NzIiwiZXhwIjoxNjQ2NDA5MjM5fQ"
+            + ".zSEbvMJedOGo16yBOXecLgucWyavnoVu023cterreUF0sxPlmV-Qu8Y7OloJUKrLGlNweUr8mVpYWPzE0iNyYw";
+    private static final String NOT_AUTHORIZATION = "Bearer notauthed";
+
+    @Mock(lenient = true)

```

Comment:
Why are you using lenient?

Reply:
It needed it before, your previous advice means I don't need it